client/elements/suttaplex/sc-suttaplex.html

```diff
@@ -351,8 +353,7 @@ <h3 class="suttaplex-heading" title="[[_getMainHeadingTitle(item, localize)]]">
                   </paper-icon-button>
                   <sc-badge class="absolute-pos behind-button" number="[[item.parallel_count]]"></sc-badge>
                 </div>
-                <paper-tooltip for="parallels_toggle" class="sc-tooltip" offset="-1" fit-to-visible-bounds
-                               position="right">
+                <paper-tooltip for="parallels_toggle" class="sc-tooltip" offset="-1">
```

Comment:
There is a reason for this "fit-to-visible-bounds". Otherwise the tooltip sometimes disappears off the right side to the screen.

Reply:
it seems that the combination of `fit-to-visible-bounds` and `position="right"` made it go haywire when the window wasn't wide enough to fit the tooltip, so I deleted both to make the tooltip appear bottom from the cursor.